spk/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,37 @@
+SPK_NAME = syncthing-inotify
+SPK_VERS = 0.8.2
+SPK_REV = 1
+SPK_ICON = src/${SPK_NAME}.png
+BETA = 1
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/busybox cross/$(SPK_NAME)
+SPK_DEPENDS = syncthing
+START_DEP_SERVICES = syncthing
+
+MAINTAINER = SynoCommunity
+DESCRIPTION  = Syncthing uses a rescan interval to detect changes in folders. Syncthing-inotify uses OS primitives to detect changes as soon as they happen. Therefore, near real-time synchronisation can be achieved.
```

Comment:
Double space between `DESCRIPTION` and `=`
Maybe start with `File watcher intended for use with Syncthing.`, then append the current text?